### ISSUE TITLE
Generating documentation throws error

### ISSUE THREAD

**Issue by stefanpearson** [google/google-api-nodejs-client#385]
```
Stefans-MacBook-Air-2:google-api-nodejs-client Stefan$ npm run generate-docs

> googleapis@1.1.5 generate-docs /Users/Stefan/Projects/google-api-nodejs-client
> jsdoc -c jsdoc-conf.json ./README.md


/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/lib/jsdoc/tutorial/resolver.js:80
        names = Object.keys(meta);
                       ^
TypeError: Object.keys called on non-object
    at Function.keys (native)
    at addTutorialConf (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/lib/jsdoc/tutorial/resolver.js:80:24)
    at addTutorialConf (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/lib/jsdoc/tutorial/resolver.js:82:13)
    at /Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/lib/jsdoc/tutorial/resolver.js:141:21
    at Array.forEach (native)
    at Object.exports.load (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/lib/jsdoc/tutorial/resolver.js:115:11)
    at Object.module.exports.cli.resolveTutorials (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/cli.js:410:18)
    at Object.module.exports.cli.processParseResults (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/cli.js:393:13)
    at module.exports.cli.main (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/cli.js:236:14)
    at Object.module.exports.cli.runCommand (/Users/Stefan/Projects/google-api-nodejs-client/node_modules/jsdoc/cli.js:192:5)

npm ERR! Darwin 14.0.0
npm ERR! argv "node" "/usr/local/bin/npm" "run" "generate-docs"
npm ERR! node v0.10.32
npm ERR! npm  v2.1.6
npm ERR! code ELIFECYCLE
npm ERR! googleapis@1.1.5 generate-docs: `jsdoc -c jsdoc-conf.json ./README.md`
npm ERR! Exit status 8
npm ERR! 
npm ERR! Failed at the googleapis@1.1.5 generate-docs script.
npm ERR! This is most likely a problem with the googleapis package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     jsdoc -c jsdoc-conf.json ./README.md
npm ERR! You can get their info via:
npm ERR!     npm owner ls googleapis
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /Users/Stefan/Projects/google-api-nodejs-client/npm-debug.log
```

Any ideas? :)

**Comment by Alaneor** [comment]
[This file](/google/google-api-nodejs-client/blob/master/examples/directory_v1/authData.json) causes jsdoc to throw. Perhaps it does not expect to see a JSON...? Perhaps we could change it to Markdown file or plain JS.

**Comment by Alaneor** [comment]
I just tried several attempts at forcing jsdoc to ignore the json file, to no avail. It seems that jsdoc completely ignores the exclusion rules. In fact, it seems it also ignores the `includePattern` - were it not, a json file would never have made its way into the parsing phase.

I will consider opening a bug/PR at jsdoc repo (maybe later today or during weekend, unless someone beats me to it!).

**Comment by JustinBeckwith** [comment]
This should be fixed now.  Please make sure to `npm install` before running the command.  Thanks!

**Comment by jmdobry** [comment]
Fixed by #522
